lib/feedback.js

```diff
@@ -1,9 +1,9 @@
 "use strict";
 var texvcjs = require("texvcjs");
 var indentifierExtractor = module.exports.treeFromAst = require('./identifier');
-var packageList = ['ams', 'cancel', 'color', 'euro', 'teubner'];
+var packageList = ['ams', 'cancel', 'color', 'euro', 'teubner', 'mhchem'];
 var tokList = require('./flatList.js');
-module.exports = function (input) {
+module.exports = function (input,chem) {
```

Comment:
It would be better to pass in an `options` object directly, instead of appending additional parameters, so that in the function you can say:

```js
options = options || {};
```